addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1054,3 +1054,76 @@ test("Canned response last used changes on posting", async () => {
     cannedResponse = pyEnv["mail.canned.response"].search_read([["id", "=", cannedResponseId]])[0];
     expect(cannedResponse.last_used).not.toBeEmpty();
 });
+
+test("Canned response suggestions are not focused by default", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    pyEnv["mail.canned.response"].create({ source: "Hello", substitution: "Hello! How are you?" });
+    await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "@");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "Mitchell Admin",
+    });
+    await insertText(".o-mail-Composer-input", ":", { replace: true });
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "HelloHello! How are you?",
+        count: 0,
+    });
+});
+
+test("Focus first canned response suggestion when pressing TAB/ARROW keys", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    pyEnv["mail.canned.response"].create([
+        { source: "Hello", substitution: "Hello! How are you?" },
+        { source: "Howdy", substitution: "Howdy! How are you?" },
+    ]);
+    await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", ":");
+    await contains(".o-mail-NavigableList-item", {
+        text: "HelloHello! How are you?",
+        before: [".o-mail-NavigableList-item", { text: "HowdyHowdy! How are you?" }],
+    });
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", { count: 0 });
+    await triggerHotkey("Tab");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "HelloHello! How are you?",
+    });
+    await triggerHotkey("Escape");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", { count: 0 });
+    await insertText(".o-mail-Composer-input", ":", { replace: true });
+    await contains(".o-mail-NavigableList-item", {
+        text: "HelloHello! How are you?",
+        before: [".o-mail-NavigableList-item", { text: "HowdyHowdy! How are you?" }],
+    });
```

Comment:
No need to repeat the ordering. We can assume it's the exact same as before as you used the same trigger.
I'd just make the minimal `contains()` required for awaiting it shows the suggestion list (so even `.o-mail-NavigableList-item {2}` is good).